File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L6/05/singlearray.py
import sys

n = [0] * 10

def main():
    for i in range(10):
        n[i] = int(input("Read a number: "))
    print_array()

def print_array():
    for i in range(10):
        print(n[9-i])

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-06/plagiarized/L4/05/singlearray.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const n: number[] = new Array(10);

function print(): void {
  for (let a = 9; a >= 0; a--) {
    console.log(n[a]);
  }
}

rl.question('Read a number: ', (num) => {
  n[0] = parseInt(num);
  for (let i = 1; i < 10; i++) {
    rl.question('Read a number: ', (num) => {
      n[i] = parseInt(num);
      if (i === 9) {
        print();
        rl.close();
      }
    });
  }
});

